sycl/include/sycl/ext/oneapi/latency_control/properties.hpp

```diff
@@ -0,0 +1,79 @@
+//==--- properties.hpp - SYCL properties associated with latency_control ---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <sycl/ext/oneapi/properties/property.hpp>
+#include <sycl/ext/oneapi/properties/property_value.hpp>
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+namespace ext {
+namespace oneapi {
+namespace experimental {
```

Comment:
Use 1-line C++17 namespace

Reply:
@pvchupin: I forget are DPC++ headers allowed to use C++17 features now, or are we still trying to support a C++14 mode?

Reply:
Okay, I will update it to C++17 1-line namespace then.